src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -36,6 +36,7 @@ const CoverImageEditModal = ({
   facility,
 }: Props) => {
   const [isUploading, setIsUploading] = useState(false);
+  const [isDeleting, setIsDeleting] = useState(false);
```

Comment:
@AnkurPrabhu 

`isDeleting` and `isUploading` can be merged into an `isProcessing` State, since it's best to block both actions, when one action is pending. 

While you're at it, can you also move out the Constants outside of the component, no reason for it to get redeclared every time the component re-renders.

Reply:
will do the changes

Reply:
@gigincg have resolved your comment